web/src/plugins/airtable.js

```diff
@@ -28,6 +28,25 @@ Airtable.install = function (Vue) {
       });
     }
   };
+
+  Vue.prototype.$topics = [];
+
+  Vue.prototype.$getTopics = function (successCallback, errorCallback) {
+    if (this.$topics.length) {
+      // return cached response
+      successCallback(this.$topics);
+    } else {
+      this.$db('Topics').select({ view: 'All' }).firstPage((error, records) => {
+        if (error) {
+          console.error(`Error fetching topics: ${error}`);
+          errorCallback(error);
+        } else {
+          this.$topics = records; // cache response
+          successCallback(records);
+        }
+      });
+    }
+  };
```

Comment:
I know this can be DRYed cause it's almost an unchanged copy+paste from the locations fetch, but just so we can release this ASAP, maybe the refactor can be done in a follow-up PR? What do you think @ann-kilzer ?

Reply:
I agree, let's get it in and we can tidy later.